diff -up freecell-solver-5.0.0/libfreecell-solver.pc.in.omv~ freecell-solver-5.0.0/libfreecell-solver.pc.in
--- freecell-solver-5.0.0/libfreecell-solver.pc.in.omv~	2019-07-28 17:02:53.575627764 +0200
+++ freecell-solver-5.0.0/libfreecell-solver.pc.in	2019-07-28 17:03:15.202513658 +0200
@@ -1,12 +1,6 @@
-prefix="@CMAKE_INSTALL_PREFIX@"
-exec_prefix=${prefix}
-libdir=${exec_prefix}/lib
-includedir=${prefix}/include
-
 Name:  Freecell Solver
 Description: Library for Solving Freecell and Similar Solitaire Games.
 Version: @VERSION@
 URL: http://fc-solve.shlomifish.org/
-Libs: -L${libdir} -lfreecell-solver
+Libs: -lfreecell-solver
 Libs.private: -lm
-Cflags: -I${includedir}
